tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleModuleProjectImporter.java

```diff
@@ -83,7 +85,23 @@ private boolean findGradleFile( File file )
     @Override
     public void importProject( String location, IProgressMonitor monitor ) throws CoreException
     {
-        GradleUtil.importGradleProject( new File( location ), monitor );
+        final File projectLocation = new File( location );
+
+        final IStatus status = GradleUtil.importGradleProject( projectLocation, monitor );
+
+        if( status.isOK() )
+        {
+            final IProject project = CoreUtil.getProject( projectLocation.getName() );
```

Comment:
@AndyWu2015 this code assumes that there is only one project in the import.  What about the case of service builder?  So can you add some new code that detects all projects that were imported?  It should probably search through the file location and look for .project files starting at the location of the original import.  And for each project, we need to get the project name (from the .project file) and then add facets to that project.